index-test.js

```diff
@@ -748,4 +748,16 @@ describe('reactElementToJSXString(ReactElement)', () => {
   </div>
 </div>`);
   });
+
+  it('should return the functionValue value when "showFunctions" is true and functionValue is a valid string', () => {
+    expect(
+      reactElementToJSXString(<div fn={() => 'value'}/>, {showFunctions: true, functionValue: 'Replace me'})
```

Comment:
Can you make it so that functionValue is a function that should return a string and gets the original function code (fn.toString()) in input? I guess that'd be more generic from the ground up.

```js
functionValue: (originalCode) => {
  return 'whatever';
};
```